### DISCUSSION TITLE
Energy Management for a 2-Motor BEV using Model-Predictive Control (246)

### DISCUSSION THREAD

**Discussion by Jishnuraj17** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#95] [Collaborate]
[Energy Management for a 2-Motor BEV using Model-Predictive Control]
(https://github.com/Jishnuraj17/Energy-Management-for-a-2-Motor-BEV-using-Model-Predictive-Control)

**Comment by robertogl** [comment]
Hi @Jishnuraj17,

Would that be your solution? In that case you would need to submit it using this [form](https://www.mathworks.com/academia/student-challenge/mathworks-excellence-in-innovation-submission-form.html?tfa_1=Energy%20Management%20for%20a%202-Motor%20BEV%20using%20Model-Predictive%20Control&tfa_2=246). I noticed there is no documentation in your repository, it would be great if you could add some to help us review the solution and other people to use it in the future. You can add info about how you solved your solution in the README file including a 'How to run' section. Thanks.
